rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -162,7 +162,7 @@ void eventLogger_logReleaseBtcRequested_release_before_papyrus_and_updateCollect
     }
 
     @Test
-    void handmade_release_before_rskip_146_185_no_events_emitted() throws IOException {
+    void release_before_rskip_146_185_no_events_emitted() throws IOException {
```

Comment:
```suggestion
    void release_before_rskip_146_no_events_emitted() throws IOException {
```

since rskip 146 is previous than 185, no need to mention the 185 if the 146 is not activated, wdyt?